<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>verifyMapTools</class>
 <widget class="QWidget" name="verifyMapTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>50</width>
    <height>83</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">verifyMapTools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolButton" name="tbnCopy">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbnSearch">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="system-search"/>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCopyFromAll">
   <property name="icon">
    <iconset theme="media-seek-backward"/>
   </property>
   <property name="text">
    <string>All files</string>
   </property>
   <property name="toolTip">
    <string>From left list</string>
   </property>
  </action>
  <action name="actionCopyFromMissing">
   <property name="icon">
    <iconset theme="media-seek-forward"/>
   </property>
   <property name="text">
    <string>Missing files only</string>
   </property>
   <property name="toolTip">
    <string>From right list</string>
   </property>
  </action>
  <action name="actionSearchFromAll">
   <property name="icon">
    <iconset theme="media-seek-backward"/>
   </property>
   <property name="text">
    <string>All files</string>
   </property>
   <property name="toolTip">
    <string>From left list</string>
   </property>
  </action>
  <action name="actionSearchFromMissing">
   <property name="icon">
    <iconset theme="media-seek-forward"/>
   </property>
   <property name="text">
    <string>Missing files only</string>
   </property>
   <property name="toolTip">
    <string>From right list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
